decidim-proposals/app/models/decidim/proposals/proposal.rb

```diff
@@ -202,7 +202,7 @@ def can_accumulate_supports_beyond_threshold
       # user - the user to check for authorship
       def editable_by?(user)
         return true if draft?
-        authored_by?(user) && !answered? && within_edit_time_limit? && !copied_from_other_component?
+        !answered? && within_edit_time_limit? && !copied_from_other_component? && created_by?(user)
```

Comment:
Only creator authors can edit a proposal then?

Reply:
@decidim/product can you answer this please?

Reply:
hi @oriolgual I confirm that only the authors can edit a proposal (having version control enabled).

Reply:
Thanks @carolromero

Reply:
@carolromero what do you mean by "having version control enabled"?

Reply:
@oriolgual There is the 4567 PR with the version control feature

Reply:
Sooo, should we add another check here to know if version control is enabled?

Reply:
@oriolgual I understand that what @carolromero has said is that control version will always be enabled (once is merged into master) and that only Amendments as a whole can be enabled/disabled.